internal/pkg/term/syncbuffer/termprinter_test.go

```diff
@@ -90,6 +90,11 @@ line4 from image2`)
 				buf.MarkDone()
 			}
 			ltp.Print()
+			if ltp.numLines == -1 {
```

Comment:
nit: can you add a comment here explaining why you're doing this :)

Instead of `ltp.numLines == -1` can we add a bool into the testcase that's like `t.isPrintAll` and then do this for-loop. I'm mentioning this because we can always change the "magic number" from -1 to something else and then the test would stop working.